### ISSUE TITLE
absolute-relative ignores settings.segments and stops reporting inside slices with custom segments

### ISSUE THREAD

**Issue by conarti** [conarti/eslint-plugin-feature-sliced#37]
## Behaviour

`absolute-relative` resolves the slice of a path with the built-in segment list only. The rule reads `settings['@conarti/feature-sliced'].layers` but never `settings['@conarti/feature-sliced'].segments`, so a custom segment name is not recognised as a segment, the current file appears to live in a different slice than the target, and the "must be relative inside one slice" check never fires.

## Reproduction

Config: `featureSliced({ segments: ['services'] })`.

File `src/entities/cart/services/a.ts`:

```ts
import { b } from 'src/entities/cart/ui/b';
```

Expected: `must-be-relative-path`, the import stays inside the `cart` slice.
Actual: no message.

The same import written from `src/entities/cart/model/a.ts` (a default segment) is reported as expected.

## Where

- `src/rules/absolute-relative/index.ts` calls `extractLayersConfig(context)` only.
- `src/rules/absolute-relative/model/validate-and-report.ts` passes `{ layersConfig }` to `extractPathsInfo` without `segmentsConfig`, so `extractSlice` falls back to `DEFAULT_SEGMENTS`.

`layers-slices` and `public-api` already pass `segmentsConfig` through, so the fix is the same wiring plus a rule case with a custom segment. Related: #35, `no-cross-segment-reexport` has the same gap.
